readme.md

```diff
@@ -396,6 +396,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Services Engineering](https://github.com/mmcgrana/services-engineering)
 - [Free for Developers](https://github.com/ripienaar/free-for-dev)
 - [Bitcoin](https://github.com/igorbarinov/awesome-bitcoin)
+- [Blockchain](https://github.com/igorbarinov/awesome-blockchain)
```

Comment:
Needs a very short description on what a blockchain is. See other description on how to format it.